### DISCUSSION TITLE
Document the gitBranch body parameter in the REST-API/Environment variables Docs

### DISCUSSION THREAD

**Discussion by giancarlosisasi** [vercel/community#243] [Help] [answered]
I was reading the [Create one or more environment variables](https://vercel.com/docs/rest-api#endpoints/projects/create-one-or-more-environment-variables) and noticed that the `gitBranch` body parameter was not documented :(

The documentation says: 
_Create one ore more environment variables for a project by passing its key, value, type and target_

We need to add the missing `gitBranch` option to the last part (the body parameters). It should be:
_Create one ore more environment variables for a project by passing its key, value, type, **gitBranch** and target_

**Comment by Pieparker** [comment]
Hey there, any chance you can elaborate on the issue or what's missing?

**Comment by giancarlosisasi** [comment, marked as answer]
Hi @Pieparker 
The documentation says: 
_Create one ore more environment variables for a project by passing its key, value, type and target_

We need to add the missing `gitBranch` option to the last part (the body parameters). It should be:
_Create one ore more environment variables for a project by passing its key, value, type, **gitBranch** and target_

**Reply by Pieparker** [reply]
Thank you; I've forwarded this to our documentation team to review!

ref: https://github.com/vercel/documentation/issues/98 (internal only)

**Reply by giancarlosisasi** [reply]
@Pieparker thank you!
